desktop/server/index.js

```diff
@@ -37,6 +37,14 @@ function showAppWindow() {
 
 	mainWindow.webContents.on( 'did-finish-load', function() {
 		mainWindow.webContents.send( 'app-config', Config, Settings.isDebug(), System.getDetails() );
+
```

Comment:
If this needs to only run once then the best place would be to move it to an app/window handler and load it from `app.js`